lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -394,7 +400,7 @@ export class ResponseHandler {
             cloudGraphHostName: cacheRecord.account?.cloudGraphHostName || Constants.EMPTY_STRING,
             msGraphHost: cacheRecord.account?.msGraphHost || Constants.EMPTY_STRING,
             code,
-            fromNativeBroker: false,
+            fromNativeBroker: accountId ? true : false,
```

Comment:
Get rid of this, this isn't a native call.

Reply:
Done